expression_json.go

```diff
@@ -243,6 +243,8 @@ func parseJsonSubexpr(data []interface{}) (Expression, error) {
 		return parseJsonLessThan(data)
 	case "greaterequal":
 		return parseJsonGreaterEquals(data)
+	case "not":
```

Comment:
This is a duplicate case statement.  Not is handled above.

Reply:
Removed